### ISSUE TITLE
missing `protocol-9p`

### ISSUE THREAD

**Issue by samoht** [fdopen/opam-repository-mingw#16]
Any reason why [protocol-9p](https://github.com/ocaml/opam-repository/tree/master/packages/protocol-9p) is not in the list?

**Comment by fdopen** [comment]
It depends on unix-errno which does not compile on windows for a for a variety of reasons. So neither unix-errno nor protocol-9p were added.

I've only recently changed the merge scripts, so nearly all upstream commits will be added to this repo, but marked as non-available on windows in a further commit, if they don't compile on windows. I will see if I can change it further to link or include the error log in the message of this commit. But the error log is often too short or misleading.  (In the case of unix-errno the log would complain about a missing ctypes dependency - but of course that's not the real reason, unix errno don't compile. The various errno codes are just defined on windows [and the c compiler is hardcoded as cc, and .... ])
